DDPFF/run/colorpcd.py

```diff
@@ -0,0 +1,25 @@
+import sys
+import numpy as np
+import open3d as o3d
+
+
+path = 'input/result.ply'
+pcd = o3d.io.read_point_cloud(path)
+
+planes = []                
+with open('output/planes.txt', 'r') as f:
+    for line in f:
+        planes.append([int(x) for x in line.split()])
+    
+colors = np.array(pcd.colors)
+
+for ind in planes:
+    col = np.random.uniform(0,1, size=(1,3))
```

Comment:
Is it possible for the same colors to be generated for the different planes?

Reply:
Done

Reply:
I think it may happen with very low probability. Because it's really hard to match three the same number again, especially when we use decimal numbers

Reply:
Without the possibility to detect such cases even low probability will be a problem, a possible solution can be found here https://github.com/DmiitriyJarosh/PlaneDetector/blob/main/src/utils/colors.py#L34 (but here color is generated in not normalized form)

Reply:
Done